buckw

```diff
@@ -213,10 +213,11 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    # Go directly to the kill command, help command, or buck options. Do not run okbuck.
+   skip=("kill" "help" "--help" "-h" "--version" "-V")
```

Comment:
Buck itself would catch the error if a command is passed in a position other than `$1`. I couldn't think of a use case where it would be risky to skip if `kill` or `help` is found in another argument.  Especially since `./buckw -V help` is valid and will run the help command but ignore the `-V`.